galaxyui/src/app/my-content/namespace-detail/namespace-detail.component.html

```diff
@@ -3,13 +3,24 @@
     <form [formGroup]="namespaceForm" novalidate>
         <div class="form-group">
             <label for="nameInput">Name <span class="required">*</span></label>
-            <input type="text"
+            <input *ngIf="!me.staff"
```

Comment:
I don't this is right. Think the formControl object should be disabled in the Javascript. See https://github.com/ansible/galaxy/blob/release/3.0/galaxyui/src/app/my-content/namespace-detail/namespace-detail.component.ts#L273